1.data-exploration/sample_info_data_visualization.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+# import the necessary packages that will be utilized
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd 
+import pathlib
+import seaborn as sns
+
+# assign the desired file a variable using pathlib.Path command
+input_file = pathlib.Path("../0.data-download/data/sample_info.csv")
+
+# set the data frame to be the desired .csv file that is read by pandas(pd) using the pd.read_csv(desired file read as a previously defined variable)
+df = pd.read_csv(input_file)
+
+# print the parameters of the read file
+print(df.shape)
+
+
+# In[2]:
+
+
+# how many samples?
+n_samples = len(df["DepMap_ID"])
+print(f"Number of Samples: {n_samples} \n")
+
+# how many genes 
+all_cancers = df["age"].unique()
+print(f"Ages sampled from: \n {all_cancers} \n")
+
+# how many different types of cancer?
+all_cancers = df["primary_disease"].unique()
+print(f"All Cancer Types: \n {all_cancers} \n")
```

Comment:
Knowing which cancer types are present is a great first step, but how many of each are there?

A bar chart for this would be great :)

Something like:

```
import plotnine as gg

data = ....LOAD DATA HERE....

(
    gg.ggplot(data, gg.aes(x="primary_disease"))
    + gg.geom_barchart()
)
```

You may also need to code a new variable called "other" in case there are many singleton cancer types.